src/components/Dashboard/Dashboard.jsx

```diff
@@ -38,19 +35,19 @@ export class Dashboard extends React.Component {
                         {(this.props.status === 'WAITING_FOR_VOTES' && this.props.voteCardDuration > 0) && <Timer cy="vote-timer" setDuration={this.props.setVoteCardTimer} duration={this.props.voteCardDuration} />}
                         <div className={styles.gameInfo}>
                             {this.props.status !== "NOT_STARTED" && <h2>Round: <span id="round-number" data-cy="round-number">{this.props.roundNum}</span></h2>}
-                            {this.props.currentWord !== '' && <h2 id="message">Word: <span data-cy='current-word'>{this.props.currentWord}</span></h2>}
                             <Players />
                         </div>
                     </div>
                     <div className={styles.middle}>
                         <div className={styles.interactions}>
-                            {showPlayerInteractions && <PlayerInteractions />}
+                            {showPImiddle && <PlayerInteractions />}
                             {(this.props.winner || this.props.drawers.length > 1) && <GameOver />}
                             {this.props.status !== "GAME_OVER" && <PlayedCards />}
                         </div>
                         {this.props.status !== "NOT_STARTED" && this.props.status !== "GAME_OVER" && <MyCards />}
                     </div>
                     <div className={styles.side}>
+                        {showPIright && <PlayerInteractions />}
```

Comment:
If these are now becoming two separate boxed then it might be worth splitting the component into two?